### ISSUE TITLE
AttributeError: module 'aiobotocore' has no attribute 'AioSession'

### ISSUE THREAD

**Issue by abhishek815** [fsspec/s3fs#642]
I'm running an AWS job and this error did not previously occur, but it has just started occurring recently. The job works perfectly fine locally, but when I run the job on AWS, the below error occurs. To reproduce the error: I am grabbing a parquet file from s3 using either pandas or dask.

 I realize this issue has occurred before on #515, but I would like to bring the issue up as the issue is occurring again.

<img width="530" alt="image" src="https://user-images.githubusercontent.com/50932225/186790362-f517cd6e-13cf-4a92-9e65-30c40fa21455.png">

**Comment by martindurant** [comment]
What version of fsspec, s3fs and aiobotocore do you have, please?

**Comment by abhishek815** [comment]
My environment does not directly have s3fs installed, but the error occurs when I call `to_parquet()` on either dask or pandas (which uses s3fs). Looking at pandas requirements file, they use the latest version of s3fs + fsspec,aiobotocore<2.0.0.

**Comment by martindurant** [comment]
can you please find out what versions you do have installed? `pip list` would help. The latest version of s3fs requires aiobotocore~=2.3.4, so something is fishy, maybe this is a pandas issue.

**Comment by raybellwaves** [comment]
Saw this on my nightly pipeline

```
aiobotocore               2.4.0
aiohttp                   3.8.1
boto3                     1.24.59
fsspec                    2021.7.0
s3fs                      2021.7.0
```

Not sure why it brought in an old version of s3fs.

**Comment by martindurant** [comment]
Oh, 2021! I didn't even notice at first. I don't know how your dependency resolution is working, best to be explicit in as many versions as you can.

**Comment by abhishek815** [comment]
Yes to be more explicit:

```
 s3fs-2022.7.1
aiobotocore~=2.3.4
fsspec-2022.7.1
```

So I should resolve this issue by explicitly instaling`aiobotocore==2.3.4`?

**Comment by raybellwaves** [comment]
> Oh, 2021! I didn't even notice at first. I don't know how your dependency resolution is working, best to be explicit in as many versions as you can.

Indeed.

Saw aiobotocore released yesterday https://github.com/aio-libs/aiobotocore/tags

Probably why my unpinned environment-dev.yml picked it up somehow yesterday

**Comment by martindurant** [comment]
fwiw, I just checked, and pip-installing pandas, dask, fastparquet and pyarrow in a fresh environment (py39) did *not* install s3fs at all, but did get fsspec 2022.7.1

**Comment by abhishek815** [comment]
Ahhh i see the issue, for now, I resolved it by adding `aiobotocore==2.3.4` to my requirements.txt file.

**Comment by raybellwaves** [comment]
Why I run my environment-dev locally (boto3, dask, s3fs) I don't get aiobotocore at all.

Nothing to do here (s3fs). For me, it was likely temporary packing resolving issues.
